.github/workflows/container-images.yml

```diff
@@ -0,0 +1,139 @@
+
+      - name: Load image
+        id: load_image
+        run: |
+          docker load -i ${{ steps.prepare.outputs.name }}-${{ steps.prepare.outputs.version }}.tar
+
+      - name: Scan image (anchore)
+        id: scan_image_anchore
+        uses: anchore/scan-action@dafbc97d7259af88b61bd260f2fde565d0668a72 # v3.3.4
+        with:
+          image: ${{ steps.prepare.outputs.name }}:${{ steps.prepare.outputs.version }}-amd64
+
+      - name: Upload SARIF (anchore)
+        id: upload_sarif_anchore
+        uses: github/codeql-action/upload-sarif@32dc499307d133bb5085bae78498c0ac2cf762d5 # v2.2.5
+        with:
+          sarif_file: ${{ steps.scan_image_anchore.outputs.sarif }}
+
+      - name: Scan image (trivy)
+        id: scan_image_trivy
+        uses: aquasecurity/trivy-action@1f0aa582c8c8f5f7639610d6d38baddfea4fdcee # v0.9.2
+        with:
+          image-ref: ${{ steps.prepare.outputs.name }}:${{ steps.prepare.outputs.version }}-amd64
+          format: sarif
+          output: trivy.sarif
+
+      - name: Upload SARIF (trivy)
+        id: upload_sarif_trivy
+        uses: github/codeql-action/upload-sarif@32dc499307d133bb5085bae78498c0ac2cf762d5 # v2.2.5
+        with:
+          sarif_file: trivy.sarif
+
+      - name: Scan SBOM
+        id: scan_sbom
+        uses: anchore/scan-action@dafbc97d7259af88b61bd260f2fde565d0668a72 # v3.3.4
+        with:
+          sbom: sbom-x86_64.spdx.json
+
+      - name: Publish SBOM
+        id: publish_sbom
+        uses: anchore/sbom-action/publish-sbom@07978da4bdb4faa726e52dfc6b1bed63d4b56479 # v0.13.3
+        with:
+          sbom-artifact-match: sbom-x86_64.spdx.json
+
+      - name: Login to GitHub Container Registry
+        if: github.ref == 'refs/heads/main'
+        id: docker_login
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a # v2.1.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Tag image
+        if: github.ref == 'refs/heads/main'
+        id: tag_image
+        run: |
+          docker tag \
+            ${{ steps.prepare.outputs.name }}:${{ steps.prepare.outputs.version }}-amd64 \
+            ghcr.io/ministryofjustice/data-platform-${{ steps.prepare.outputs.name }}:${{ steps.prepare.outputs.version }}
+          docker tag \
+            ${{ steps.prepare.outputs.name }}:${{ steps.prepare.outputs.version }}-amd64 \
+            ghcr.io/ministryofjustice/data-platform-${{ steps.prepare.outputs.name }}:latest
+
+      - name: Push image
+        if: github.ref == 'refs/heads/main'
+        id: push_image
+        run: |
+          docker push ghcr.io/ministryofjustice/data-platform-${{ steps.prepare.outputs.name }}:${{ steps.prepare.outputs.version }}
+          docker push ghcr.io/ministryofjustice/data-platform-${{ steps.prepare.outputs.name }}:latest
+
+      - name: Sign image
```

Comment:
The use of ghcr feels not so tactical at this point.

Reply:
🐳 will move to DockerHub